### ISSUE TITLE
Feature request: set project default easing interpolation setting

### ISSUE THREAD

**Issue by cyaoeu** [hidefuku/AnimeEffects#15]
It would be useful if you could set a project default easing interpolation setting in case you don't like linear interpolation (who does?). This setting would then be used for newly created keyframes.

Or any other method really like changing easing settings for multiply keyframes at once.

**Comment by 2blackbar** [comment]
I agree, that would be nice, or selecting multiple keyframes and changing interpolation for all of them at once
